docs/src/user-guide/formatters/index.md

```diff
@@ -83,28 +97,48 @@ function addOne(i) {
 ```
 
 ### json
 
+TODO: quick overview
```

Comment:
This one just outputs JSON-serialized lint results. Useful when you want to do something with it programmatically, because it's structured data.

Context: https://github.com/eslint/eslint/issues/3036